work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -221,29 +221,60 @@ def specify_file(self, local_name, remote_name=None, type=None, flags=None, cach
         return work_queue_task_specify_file(self._task, local_name, remote_name, type, flags)
 
     ##
-    # Add a file to the task which will be transfered with a command at the worker.
+    # Add a url to the task which will be provided as an input file.
     #
     # @param self           Reference to the current task object.
+    # @param url            The url of the file to provide.
     # @param remote_name    The name of the file as seen by the task.
-    # @param cmd            The shell command to transfer the file. Any
-    #                       occurance of the string %% will be replaced with the
-    #                       internal name that work queue uses for the file.
     # @param type           Must be one of the following values: @ref WORK_QUEUE_INPUT or @ref WORK_QUEUE_OUTPUT
     # @param flags          May be zero to indicate no special handling, or any
     #                       of the @ref work_queue_file_flags_t or'd together The most common are:
     #                       - @ref WORK_QUEUE_NOCACHE (default)
     #                       - @ref WORK_QUEUE_CACHE
-    #                       - @ref WORK_QUEUE_WATCH
-    #                       - @ref WORK_QUEUE_FAILURE_ONLY
     # @param cache         Whether the file should be cached at workers (True/False)
-    # @param failure_only  For output files, whether the file should be retrieved only when the task fails (e.g., debug logs).
     #
     # For example:
     # @code
-    # # The following are equivalent
-    # >>> task.specify_file_command("my.result", "chirp_put %% chirp://somewhere/result.file", type=WORK_QUEUE_OUTPUT)
+    # >>> task.specify_url("http://www.google.com/","google.txt",type=WORK_QUEUE_INPUT,flags=WORK_QUEUE_CACHE);
+    # @endcode
+
+    def specify_url(self, url, remote_name, type=None, flags=None, cache=None, failure_only=None):
```

Comment:
type might not be needed. Code seems only to consider input files.

Reply:
Showing type for consistency with other file specifications.  Throws ValueError if WORK_QUEUE_OUTPUT is given.